include/seqan3/argument_parser/validators.hpp

```diff
@@ -256,17 +257,38 @@ class file_ext_validator
 
     /*!\brief Constructing from a vector.
      * \param[in] v The vector of valid file extensions to test (e.g. {"fa", "fasta"}).
+     * \param[in] c Case sensitivity flag. Set true for case sensitivity.
+     *              Default: false (case insensitive).
+     *              For case insensitivity, the initializer list is transformed to lower case characters.
```

Comment:
this can go into details :)

```
     * \param[in] c Compare file extensions case sensitive or not. Default: false (case insensitive).
     *
     * For case insensitivity, everything is converted to lower case characters.
```